src/App.js

```diff
@@ -12,13 +13,110 @@ const goodsFromServer = [
   'Honey',
   'Jam',
   'Garlic',
-];
+].map((item, index) => (
+  // eslint-disable-next-line no-param-reassign
+  item = {
+    id: index,
+    nameGoods: item,
+  }
+));
```

Comment:
з map можна зразу повертати об'єкт щоб не робити присвоєння в item
```suggestion
].map((item, index) => ({
    id: index,
    nameGoods: item,
  }
));
```